.github/workflows/main.yml

```
@@ -215,61 +215,44 @@ jobs:
     - run: cargo test -p wasm-bindgen-futures
     - run: cargo test -p wasm-bindgen-shared
 
-  test_web_sys:
-    name: "Run web-sys crate tests"
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v4
-    - run: rustup update --no-self-update stable && rustup default stable
-    - run: rustup target add wasm32-unknown-unknown
-    - uses: actions/setup-node@v4
-      with:
-        node-version: '20'
-    - uses: ./.github/actions/setup-geckodriver
-    - run: cargo build --manifest-path crates/web-sys/Cargo.toml --target wasm32-unknown-unknown
-    - run: cargo build --manifest-path crates/web-sys/Cargo.toml --target wasm32-unknown-unknown --features Node
-    - run: cargo build --manifest-path crates/web-sys/Cargo.toml --target wasm32-unknown-unknown --features Element
-    - run: cargo build --manifest-path crates/web-sys/Cargo.toml --target wasm32-unknown-unknown --features Window
-
-  test_web_sys_all_features:
-    name: "Run web-sys crate tests with all features"
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v4
-    - run: rustup update --no-self-update stable && rustup default stable
-    - run: rustup target add wasm32-unknown-unknown
-    - uses: actions/setup-node@v4
-      with:
-        node-version: '20'
-    - uses: ./.github/actions/setup-geckodriver
-    - run: cargo test --manifest-path crates/web-sys/Cargo.toml --target wasm32-unknown-unknown --all-features
-
-  test_web_sys_all_features_unstable:
-    name: "Run web-sys crate tests with all features and unstable APIs"
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v4
-    - run: rustup update --no-self-update stable && rustup default stable
-    - run: rustup target add wasm32-unknown-unknown
-    - uses: actions/setup-node@v4
-      with:
-        node-version: '20'
-    - uses: ./.github/actions/setup-geckodriver
-    - run: cargo test --manifest-path crates/web-sys/Cargo.toml --target wasm32-unknown-unknown --all-features
-      env:
-        RUSTFLAGS: --cfg=web_sys_unstable_apis
-
-  check_web_sys:
-    name: "Verify that web-sys is compiled correctly"
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v4
-    - run: rustup update --no-self-update stable && rustup default stable
-    - run: cd crates/web-sys && cargo run --release --package wasm-bindgen-webidl -- webidls src/features ./Cargo.toml
-    - run: git diff --exit-code
-
-  test_js_sys:
-    name: "Run js-sys crate tests"
+  test_node:
```

```suggestion
  test_crates:
```
Can't come up with anything better.

---

These all use the webdriver so maybe `test wasm-bindgen with webdriver (test name)`?